### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-01-23

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#655]
## Time

**UTC Wed 23-Jan-2019 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 23-Jan-2019 08:00 (08:00 AM) |
| US / Mountain | Wed 23-Jan-2019 09:00 (09:00 AM) |
| US / Central  | Wed 23-Jan-2019 10:00 (10:00 AM) |
| US / Eastern  | Wed 23-Jan-2019 11:00 (11:00 AM) |
| London        | Wed 23-Jan-2019 16:00 (04:00 PM) |
| Amsterdam     | Wed 23-Jan-2019 17:00 (05:00 PM) |
| Moscow        | Wed 23-Jan-2019 19:00 (07:00 PM) |
| Chennai       | Wed 23-Jan-2019 21:30 (09:30 PM) |
| Hangzhou      | Thu 24-Jan-2019 00:00 (12:00 AM) |
| Tokyo         | Thu 24-Jan-2019 01:00 (01:00 AM) |
| Sydney        | Thu 24-Jan-2019 03:00 (03:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2019-01-23&iso=20190123T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Jan+23%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/14idQQcRRyzlMYRbBJH_VHGAde7Qu369Biy0nDxo1eYM/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* repl: emit uncaughtException [#20803](https://github.com/nodejs/node/pull/20803)

### nodejs/TSC

* NODE_MODULE_VERSION for Electron Major Releases [#651](https://github.com/nodejs/TSC/issues/651)
* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* James M Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by Trott** [comment]
No agenda items, so seems like a high likelihood of cancelling.

Two things it is worth looking over and preparing for in general are https://github.com/nodejs/node/pull/22584 and https://github.com/nodejs/node/pull/21351. One or both will almost certainly will come before TSC before the April release. They are controversial things with long histories. It would be great if, when the inevitable discussion occurs, everyone who is participating sounds prepared.

I will likely miss the first 15+ minutes, but can attend otherwise.

**Comment by mcollina** [comment]
There is a security issue we should talk about.

**Comment by Trott** [comment]
For the NODE_MODULE_VERSION ABI issue (https://github.com/nodejs/TSC/issues/651), perhaps invite @codebytere to attend as a guest? Maybe after confirming that there will be sufficient attendance by invested/informed folks? Maybe folks can self-nominate on that front, but judging from the issues and previous conversations, likely candidates might be @gabrielschulhof @mhdawson @mcollina @MylesBorins @ofrobots @rvagg

**Comment by mhdawson** [comment]
@gabrielschulhof @mhdawson @mcollina @MylesBorins @ofrobots @rvagg can you chime in if you can make it.  If enough can we can see if @codebytere can make it.

**Comment by rvagg** [comment]
Won't be making it, 3am, but for the `NODE_MODULE_VERSION` discussion, our docs state that if a third party wants to reserve a number they need to open an issue here, as has been done in the recent past for Debian. I don't think there's any blockers to just saying _sure, take what you need!_ Not sure we even need to do it in a meeting really, opening an issue ought to be enough for an embedder or extender.

But we need to keep track of these reservations; hence my proposal for a registry @ https://github.com/nodejs/node/pull/24114 - one central place to keep a record, machine readable and easy to PR a request to reserve for Electron and others that need a number. Incrementing is cheap, there shouldn't be a reason to block anyone requesting as long as we can do it in a coordinated fashion. I haven't forgotten about that PR, it's just been a low priority and there was a good suggestion for more metadata that I was considering changing the format to incorporate.

**Comment by apapirovski** [comment]
I don't think https://github.com/nodejs/node/pull/20803 should be on the agenda. It looks like the request was for TSC to review the PR since it's semver-major. I do not think there is anything to discuss at the meeting...

**Comment by BridgeAR** [comment]
@apapirovski I agree that it's not really sonething to discuss. I only added it because no one pointed it in any direction and that seems pretty bad. It would be good to get any response. Yes, there were a few comments but no one either gave a LG or said anything against landing it.

For that reason I kindly ask the TSC members to review more semver-major PRs as it's otherwise difficult to deal with these.

**Comment by ofrobots** [comment]
I can't make it on this timeslot unfortunately.

**Comment by thefourtheye** [comment]
Sorry for the delay. No significant moderation team activities this week.

---

cc @nodejs/tsc @nodejs/community-committee @nodejs/moderation

**Comment by Trott** [comment]
Minutes PR is https://github.com/nodejs/TSC/pull/658 so I think this can be closed.
